### PR TITLE
feat(sessions): :lipstick: new design proposition

### PR DESCRIPTION
**Pull request by gouz**
New proposal for a better readability on desktop

image

### CODE REVIEW

**Comment by gouz** [comment]
Not the best solution I made to fill the gap, but it works

File: _includes/layouts/sessions/list.11ty.js [hunk 1]
```diff
@@ -4,108 +4,154 @@
...
+  let rooms = [];
+  Object.keys(data.site[data.locale].rooms).forEach((r) => {
+    const room = data.site[data.locale].rooms[r];
+    const sess = sessions.filter((session) => session.data.room == r);
+    if (sess.length)
+      rooms.push(`
+    <section class="room-tracklist ${r}">
+      <p class="room">${room.name}</p>
+      <section class="grid gap">
+        ${
```
Comment (gouz) [inline _includes/layouts/sessions/list.11ty.js:88]: This is the part I'm not proud of ^^

**Comment by LostInBrittany** [comment]
What about the responsivity? Does it works well on tablet and mobile?

**Comment by gouz** [comment]
Yes, I keep the current design, plus name of rooms

**Comment by gouz** [comment]
> What about the responsivity? Does it works well on tablet and mobile?

Yes, I keep the current design, plus name of rooms

**Comment by scraly** [comment]
In my side on firefox in my mac m1 I see this rendering:
image

I know, it's not easy easy but is it possible to have a render like this:
https://sunny-tech.io/schedule

? 😊 

Thx!

**Comment by gouz** [comment]
> In my side on firefox in my mac m1 I see this rendering:
> image
> 
> I know, it's not easy easy but is it possible to have a render like this:
> https://sunny-tech.io/schedule
> 
> image
> 
> ? 😊 
> 
> Thx!

Ok, I'll do it this week

**Comment by gouz** [comment]
@scraly Modifications are done ;)

**Review by scraly — APPROVED** [review]
merciii

**Pull request merged**